### ISSUE TITLE
inline with support for visible fields on detail only

### ISSUE THREAD

**Issue by chorton** [kirschbaum-development/nova-inline-relationship#19]
Hello,

I have a resource with some fields that I have a chained exceptOnForms method.  When I reference that resource from another resource using, for example, HasMany....->inline(), those fields I've set to show exceptOnForms never appear on the detail page for the resource that references the inline resource.

Is there a different way to get those fields to show up?  I just don't want those fields to appear on forms.

**Comment by brandonferens** [comment]
@chorton If I am understanding you correctly, you have for example, three field: `name`, `age`, and `email`. You don't want `age` to show up on any form, but when displaying the form via `inline()`, the visibility for `age` is not respected?

**Comment by brandonferens** [comment]
@chorton We have fixed this issue in PR #28 and will release as a new tagged version soon.

**Comment by Muetze42** [comment]
The default functions like `->exceptOnForms()` missing in this package.
Check in fields the request and you can make custom Fields with `->canSee()` and the request

Request Check:
```php
public function fields(Request $request)
    {
        Log::debug(print_r($request->all(), true));
```

Example: Hide on forms:
```php
            HasMany::make(__('Raffles'), 'raffle', Raffle::class)->inline()
                ->canSee(function () use ($request) {
                    return !$request->get('editing');
                }),
```

Example only for forms should be:
```php
            HasMany::make(__('Raffles'), 'raffle', Raffle::class)->inline()
                ->canSee(function () use ($request) {
                    return $request->get('editing');
                }),
```
